### PR TITLE
skeleton react native project

### PR DESCRIPTION
**Pull request by nataliagourova**
This PR creates a skeleton non-expo React Active Android App.
To create the skeleton, I followed instructions [here](https://reactnative.dev/docs/environment-setup) `React Native CLI Quickstart`
The code was tested via Android Studio Emulator.

To TEST, pull the code, ensure that you update android/local.properties with the path for your SDK location, properly set up the dependencies and the environment variables as described [here](https://reactnative.dev/docs/environment-setup) 
Run the app by navigating to the appropriate folder, and in two separate terminal windows run two commands: `npx react-native start` and `npx react-native run-android`

*Double check I did not commit the build and other files that need to be in the repo - I used the standard .gitignore content*

Please make sure you use the correct Android SDK path. You can find the actual location of the SDK in the Android Studio "Preferences" dialog, under Appearance & Behavior → System Settings → Android SDK.

### CODE REVIEW

**Review by CalebMcOlin — CHANGES_REQUESTED** [review]
Nice first step! Thanks for taking this task on.

Before I can really dive into reviewing this, we need to implement some `.gitIgnore` files. These will be use to keep modules, build files, and sdk keys off our repo. Doing this will also significantly reduce the amount of code we have to work through/review.

- [ ] The first `.gitIgnore` should be placed at the root level and contain:
```
node_modules
.DS_Store
```

- [ ] The second `.gitIgnore` should be placed inside the `android` folder and contain:
```
*.iml
.gradle
**/build
**.idea/caches
**.idea/libraries
**.idea/modules.xml
**.idea/workspace.xml
**.idea/tasks.xmL
local.properties
```

- [x] The third `.gitIgnore` should be placed inside the `android/app` folder and contain:
```
build
release
```

Once these are added the number of files in the PR should be **immensely smaller**. 

We can then talk about the IOS folder. Since we aren't going to be developing an IOS version at this time, we might want to remove it completely. I'm not sure what `.gitIngore` files are needed with that and it will just be cluttering up our repo/install/review process. (we can always add it back later if we are going to build an IOS version).

**Comment by nataliagourova** [comment]
@CalebMcOlin ah, I see what happened, when I copied my original local folder into the cloned repo, .gitignore did not copy. Apologies for a humongous commit!

**Comment by nataliagourova** [comment]
I added ios/ to the root .gitignore

**Review by CalebMcOlin — CHANGES_REQUESTED** [review]
No worries @nataliagourova, we've all committed changes without a `.gitIgnore` before. 

Nice job at added them back in. We still have one more step to finish cleaning this PR up. 
A `.gitIgnore` only ignores _untracked_ files, since everything was already committed they are all _tracked_ right now. This means even though the `.gitIgnore`s are added, the files are still showing up in the repo and not being ignored.

To fix this we need to remove all ignored files. I believe `git rm -r --cached .` will be our solution for this.

Here is a [StackOverflow question](https://stackoverflow.com/questions/1139762/ignore-files-that-have-already-been-committed-to-a-git-repository) about this exact issue.

**Review by dagmawidemissie17 — COMMENTED** [review]
Nice Job Natalia!
I was able to run it successfully. Caleb Left a useful review too.
Also, I notice this on the PR description
image

**Comment by nataliagourova** [comment]
@CalebMcOlin thanks for pointing me to the StackOverflow article! I pushed the changes to the branch, pulled into a local temp folder, and it looks like all is in good shape, cache cleared and .gitignore files deleted.

File: android/app/build.gradle [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+ *   // https://reactnative.dev/docs/performance#enable-the-ram-format
+ *   bundleCommand: "ram-bundle",
+ *
+ *   // whether to bundle JS and assets in debug mode
+ *   bundleInDebug: false,
+ *
+ *   // whether to bundle JS and assets in release mode
+ *   bundleInRelease: true,
+ *
+ *   // whether to bundle JS and assets in another build variant (if configured).
```
Comment (dagmawidemissie17) [inline android/app/build.gradle:32]: There appears to be a lot of comments on these files. I'm not sure if that's the deal on our project but we may be able to do a refactor sometime to get our code trimmed and neat.

**Review by dagmawidemissie17 — APPROVED** [review]
LGTM!

**Comment by nataliagourova** [comment]
@CalebMcOlin please approve if all looks good

File: android/app/build.gradle (hunk 1 above)
Comment (CalebMcOlin) [reply android/app/build.gradle:32]: I like this Idea. Do you want to create a trimming/code clean up Issue that we can put on the board?

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+*.xccheckout
+*.moved-aside
+DerivedData
+*.hmap
+*.ipa
+*.xcuserstate
+
+# Android/IntelliJ
+#
+build/
```
Comment (CalebMcOlin) [inline .gitignore:26]: @dagmawidemissie17 When you create the "clean up" Issue, add a mention to this `build/` in the root `.gitIgnore` as this is a duplicate and in this file twice.

**Comment by CalebMcOlin** [comment]
We need to create a new issue to update our README with installation steps as well as steps of how to run the app locally.

**Review by CalebMcOlin — APPROVED** [review]
Tested and works fine. Good Job!
You can "Squash and Merge" now. 🎉

**Comment by nataliagourova** [comment]
> We need to create a new issue to update our README with installation steps as well as steps of how to run the app locally.

done (not yet with a template)

File: android/app/build.gradle (hunk 1 above)
Comment (dagmawidemissie17) [reply android/app/build.gradle:32]: Sure!

**Pull request merged**